doc/conf.py

```diff
@@ -461,9 +461,45 @@ def notebook_modification_function(notebook_content, notebook_filename):
     # sklearn
     "LeaveOneOut": "sklearn.model_selection.LeaveOneOut",
     "Transformer": "sklearn.base.TransformerMixin",
-    # joblib
+    "HashingVectorizer": "sklearn.feature_extraction.text.HashingVectorizer",
+    "CountVectorizer": "sklearn.feature_extraction.text.CountVectorizer",
+    "_VectorizerMixin": "sklearn.feature_extraction.text._VectorizerMixin",
+    "StandardScaler": "sklearn.preprocessing.StandardScaler",
+    "KMeans": "sklearn.cluster.KMeans",
+    "ColumnTransformer": "sklearn.compose.ColumnTransformer",
+    "OneHotEncoder": "sklearn.preprocessing.OneHotEncoder",
+    "Pipeline": "sklearn.pipeline.Pipeline",
+    "GridSearchCV": "sklearn.model_selection.GridSearchCV",
+    "fetch_openml": "sklearn.datasets.fetch_openml",
+    # other libraries
     "joblib.Parallel": "joblib.Parallel",
     "joblib.delayed": "joblib.delayed",
+    "joblib.parallel_backend": "joblib.parallel_backend",
+    "DataFrame": "pandas.DataFrame",
+    "ndarray": "numpy.ndarray",
+    "pathlib.Path": "pathlib.Path",
+    "float64": "numpy.float64",
+    "RandomState": "numpy.random.RandomState",
+    "Series": "pandas.Series",
+    "pandas.Index": "pandas.Index",
+    "read_csv": "pandas.read_csv",
+    "pandas.merge": "pandas.merge",
+    # Skrub
+    "fetch_ken_table_aliases": "skrub.datasets.fetch_ken_table_aliases",
+    "fetch_ken_types": "skrub.datasets.fetch_ken_types",
+    "fetch_ken_embeddings": "skrub.datasets.fetch_ken_embeddings",
+    "fuzzy_join": "skrub.fuzzy_join",
+    "FeatureAugmenter": "skrub.FeatureAugmenter",
+    "GapEncoder": "skrub.GapEncoder",
+    "MinHashEncoder": "skrub.MinHashEncoder",
+    "SimilarityEncoder": "skrub.SimilarityEncoder",
+    "DatetimeEncoder": "skrub.DatetimeEncoder",
+    "GapEncoderColumn": "skrub._gap_encoder.GapEncoderColumn",
```

Comment:
I don't think that it is documented in the public API, so I am not sure that we want to add it here.
```suggestion
```